client/e2e/workflows/bceidbusiness/industry_user.spec.ts

```diff
@@ -0,0 +1,222 @@
+// 🧪 Suite to test the bceidbusiness new user workflow using storageState
+// 🔍 Asserts new user is redirected to profile
+
+import { test, expect, APIResponse } from "@playwright/test";
+// 🪄 Page Object Models
+import { DashboardPOM } from "@/e2e/poms/dashboard";
+// ℹ️ Environment variables
+import * as dotenv from "dotenv";
+import { OperatorPOM } from "@/e2e/poms/operator";
+import { fieldsUpdate } from "@/e2e/utils/helpers";
+import { deleteUserOperatorRecord } from "@/e2e/utils/queries";
+dotenv.config({ path: "./e2e/.env.local" });
+const happoPlaywright = require("happo-playwright");
+
+test.beforeEach(async ({ context }) => {
+  await happoPlaywright.init(context);
+  let response: APIResponse = await context.request.get(
+    "http://localhost:8000/api/registration/test-setup",
+  );
+  // Wait for the response and check for success status text and code (e.g., 200)
+  expect(await response.text()).toBe("Test setup complete.");
+  expect(response.status()).toBe(200);
+
+  await deleteUserOperatorRecord(process.env.E2E_INDUSTRY_USER_GUID as string);
+});
+
+test.afterEach(async () => {
+  await happoPlaywright.finish();
+});
+
+// 🏷 Annotate test suite as serial
+test.describe.configure({ mode: "serial" });
+test.describe("Test Workflow industry_user", () => {
+  // 👤 run test using the storageState for this role
+  const storageState = process.env.E2E_INDUSTRY_USER_STORAGE;
+  // Note: specify storageState for each test file or test group, instead of setting it in the config. https://playwright.dev/docs/next/auth#reuse-signed-in-state
+  test.use({ storageState: storageState }); // this will error if no such file or directory
+
+  test("Test Redirect to Dashboard", async ({ page }) => {
+    // 🛸 Navigate to dashboard page
+    const dashboardPage = new DashboardPOM(page);
+    await dashboardPage.route();
+    // 🔍 Assert that the current URL ends with "(authenticated)/dashboard"
+    await dashboardPage.urlIsCorrect();
+    const pageContent = page.locator("html");
+    await happoPlaywright.screenshot(dashboardPage.page, pageContent, {
+      component: "Industry User Dashboard page",
+      variant: "industry_user",
```

Comment:
Nice, I was thinking we should use the variant for the roles and checked here to see what you were doing. I will update the dashboard screenshots fix I merged in to use this format in my industry_user_admin e2e PR.

Reply:
Roles make sense as the variant for stuff like login pages and dashboards. For forms I've been using variants like `with errors, filled, readonly, etc.

Reply:
Okay that makes sense, I will use that format for my form screenshots thanks.